### DISCUSSION TITLE
Welcome to Getting-started-in-Technical-Writing Discussions!

### DISCUSSION THREAD

**Discussion by Bennykillua** [Bennykillua/Getting-started-in-Technical-Writing#19] [Announcements]
## 👋 Welcome!

<img src=https://octodex.github.com/images/manufacturetocat.png alt=celebrate width=300 align=right>

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas. 
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abdulrahmoncodes** [comment]
Hi, My name is Soliu, a frontend developer based in Nigeria. I'm currently honing my coding skills and learning React.js. So, I'm looking to delve into technical writing to learn more and grow in the tech niche and network with other developers
